### PR TITLE
Super calls

### PR DESCRIPTION
**Pull request by RayVR**
This resolves Issue #448 by replacing **init** calls on parent class with super(parent, self).**init**(...) tests run as far as they do on Master but neither completed.

### CODE REVIEW

File: pymc/distributions/distribution.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def __getnewargs__(self):
 
     @classmethod
     def dist(cls, *args, **kwargs):
+        #super(Distribution, cls).__new__(
```
Comment (twiecki) [inline pymc/distributions/distribution.py:30]: Left-over?

**Comment by twiecki** [comment]
Looks good otherwise.

**Comment by jsalvatier** [comment]
I'm going to leave this up to you Thomas.

**Comment by twiecki** [comment]
Thanks @jsalvatier. Main selling point for me is consistency.

@RayVR Once you respond to that comment we can merge this.

**Comment by RayVR** [comment]
You are correct that the commented line should be removed. I am happy to remove it but won't have time until tonight or tomorrow.

**Comment by twiecki** [comment]
Sounds good.

**Comment by twiecki** [comment]
Merged with 0383f07a7826ea0a71226724f405d9aaefe493a0. Thanks!

**Pull request closed without merging**